tensorflow-ops/tests/OpsTest.hs

```diff
@@ -33,25 +33,31 @@ import qualified TensorFlow.Nodes as TF
 import qualified TensorFlow.Ops as TF
 import qualified TensorFlow.Session as TF
 import qualified TensorFlow.Tensor as TF
+import qualified TensorFlow.Types as TF
 
 -- | Test that one can easily determine number of elements in the tensor.
+testSize :: Test
 testSize = testCase "testSize" $ do
-    x <- eval $ TF.size (TF.constant [2, 3] [0..5 :: Float])
+    x <- eval $ TF.size (TF.constant (TF.Shape [2, 3]) [0..5 :: Float])
```

Comment:
OverloadedLists play poorly in post 7.10 world (forcing ugly type signatures)[https://github.com/tensorflow/haskell/blob/master/tensorflow-mnist/tests/ParseTest.hs#L145]. I started taking them out opportunistically.